modules/gitops-creating-rolloutmanager-custom-resource.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/argo-rollouts-for-progressive-delivery-of-deployments.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-creating-rolloutmanager-custom-resource_{context}"]
+= Creating RolloutManager custom resource
+
+To manage progressive delivery of deployments by using the Argo Rollouts feature in {gitops-title}, you must create and configure a RolloutManager custom resource (CR) in the namespace of your choice. By default, any new `argo-rollouts` instance has permission to manage resources only in the namespace where it is deployed. But you can use Argo Rollouts in multiple namespaces as required.
+
+.Prerequisites
+
+* {gitops-title} 1.9.0, or a later version, is installed in your cluster.
+
+.Procedure 
+
+. Log in to the {product-title} web console. 
+
+. Click *Operators* -> *Installed Operators*.
+
+. Create or select the project where you want to create and configure a RolloutManager custom resource (CR) from the *Project* drop-down menu.
+
+. Select *OpenShift GitOps Operator* from the installed operators.
+
+. In the *Details* tab, under the *Provided APIs* section, click *Create instance* in the *RolloutManager* pane.
+
+. On the *Create RolloutManager* page, select the *YAML view* and use the default YAML or edit it as per your requirements:
++
+.Example: RolloutManager CR
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: RolloutManager
+metadata:
+  name: argo-rollout
+  labels:
+    example: basic
+spec: {}
+----
+
+. Click *Create*.
+
+. In the *RolloutManager* tab, under the *RolloutManagers* section, verify that the *Status* field of the RolloutManager instance shows as *Phase: Available*.
```

Comment:
object formatting https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting

Reply:
@abrennan89 Here it is not the API object but an instance of RollloutManager in the UI. Similar to an ArgoCD instance in the UI. SO we have to treat it the same way as we write for an ArgoCD instance. So I am leaving it as is.